src/lib/sedlexing.ml

```diff
@@ -585,37 +613,45 @@ module Utf16 = struct
       aux apos len
   end
 
-  let from_gen s opt_bo = from_gen (Helper.from_gen opt_bo s)
+  let min_bytes_per_uchar = 2
+  let max_bytes_per_uchar = 4
```

Comment:
Constants with the same name are also defined above but with a different value for the min. Is that intended? I so, the name should be different to clarify the context of each constant. If not, they should be factored out.

Reply:
This is for utf16 (inside the utf16 module). the previous version is for utf8 (inside the utf8 module)